.mvn/jvm.config

```diff
@@ -9,3 +9,4 @@
 --add-exports=jdk.compiler/com.sun.tools.javac.util=ALL-UNNAMED
 --add-opens=jdk.compiler/com.sun.tools.javac.code=ALL-UNNAMED
 --add-opens=jdk.compiler/com.sun.tools.javac.comp=ALL-UNNAMED
+--add-opens=java.base/java.nio=ALL-UNNAMED
```

Comment:
Why is this needed?

Reply:
According to doc:
```
Java modules are compatible with JDK 8 and above. Currently, JDK 8, 11, 17, and 18 are tested in CI.

When using Java 9 or later, some JDK internals must be exposed by adding --add-opens=java.base/java.nio=ALL-UNNAMED to the java command
```

https://arrow.apache.org/docs/java/install.html

Reply:
Unfortunately, we can’t take this change as is. It puts a burden on anyone running Trino to add those options in their configuration. Also, those “jdk internals” can change at any time and make it harder or impossible to upgrade to a new version of the JDK.

You’ll need to wait until the dependencies update to use public APIs in the JDK and be compatible with Java 17 out of the box.

Reply:
Since this feature is behind toggle, we can leave the JVM config as it is and only provide it for the arrow tests only and document it as the prerequisite for using arrow serialization.

As for the background: in our testing, arrow serialization is much, much faster than avro for consuming BigQuery datasets (difference is ~30-40%). We would like to have it already implemented and tested and in the future when Arrow/netty supports JDK 17 we will make it a default serialization method.